Adición del diseño para el submódulo de visualización de Resultados

JaimeBorregoConde opened the pull request:
Cambios realizados:
- [x] Página principal para visualización de resultados
- [x] Modificación del panel de administradores (Panel de Control Global) 
- [x] Modificación de paneles individuales de resumen de votaciones
- [x] Arreglo de los test para que funcionen con el nuevo diseño

Cambios secundarios:
- [x] Corrección de algunas erratas en el documentos de descarga de estadísticas
- [x] El botón de acceso al bot de Telegram ahora está en el menú principal
- [x] Se han añadido etiquetas de traducción para poder traducirse de una manera más cómoda

Nota: Para diferenciar módulos, se ha usado el color rojo para todo el módulo de visualización

javi-11 committed:
FEAT/Finished feature170

Modified the dashboard views.py to add d'hondt visualization

anac29 committed:
REFACTOR: Integrate changes

After integrating new features from postprocesing it was necesary to refactor change some parts of the code in order to make everythong work correctly.

javi-11 committed:
Merge pull request #171 from egc-decide-trebujena/feature/170

FEAT/Finished feature170

anac29 committed:
Merge pull request #175 from egc-decide-trebujena/feature/172

REFACTOR: Integrate changes

anac29 committed:
TEST:Created 6 test from dashboard

It was necessary to test the dashboard. Some changes have to be done to dashbard due to some error that were found developing the tests.

anac29 committed:
TEST: 6 test for dashboard

6 test were develop and some parts of the code needed to be fixed.

javi-11 committed:
FEAT/Finished feature 177

Modified the views and template of dashboard to add parity graphs and new fields

javi-11 committed:
Merge pull request #177 from egc-decide-trebujena/feature/077

Feature/077

anac29 committed:
Merge pull request #179 from egc-decide-trebujena/feature/177

FEAT/Finished feature 177

javi-11 committed:
FEAT/Progress on feature 184

Started several tests on the dashboard view

javi-11 committed:
FEAT/Progress on feature 184

Finally managed to vote through the test

javi-11 committed:
FEAT/Progress on feature 184

Finished 7 tests with more than 90% coverage and dhondt questions on dashboard

javi-11 committed:
FEAT/Finished feature 184

Finished feature 184, with 7 tests done and using selenium in almost every test

anac29 committed:
REFACTOR: A file has to be added

A file was needed in order to run dashboard tests.

anac29 committed:
Merge pull request #193 from egc-decide-trebujena/feature/184

FEAT/Added dashboard tests

JaimeBorregoConde committed:
Merge branch 'module/visualizacion' into feature/021

JaimeBorregoConde committed:
Feat: First desing for dashboard module

Principal menu added. Changes in dashboard board for improve usability.

JaimeBorregoConde committed:
Feat: First desing for unique dashboard

Change html page for indiviudal dashboard, to make it responsibe and understable.

JaimeBorregoConde committed:
Feat: Scroll Bar for main dashboard

Added scrolball for controlling big amount of data in dashboard.

JaimeBorregoConde committed:
fix: Fix test and some problems with canvas

Test has been modified to work with new desing. Fix some error with vue properties for a canvas label

JaimeBorregoConde committed:
fix: Fix sonarcloud issues
Some changes in dashboard for aproving sonarcloud issues

gonmarfer2 commented:
Considera corregir los problemas de código de Codacy antes de que pueda aceptar la pull request

gonmarfer2 commented:
- [ ] Quita por favor el print de base/tests.py
- [ ] Has subido un record.pdf que no creo que deba estar en el repositorio
- [ ] El archivo html "decide/dashboard/files/record" no tiene etiquetas de traducción
- [ ] En el archivo html "decide/dashboard/templates/dashboard/dashboard.html" faltan etiquetas de traducción

JaimeBorregoConde commented:
> * [x]  Quita por favor el print de base/tests.py
> * [x]  Has subido un record.pdf que no creo que deba estar en el repositorio
> * [x]  El archivo html "decide/dashboard/files/record" no tiene etiquetas de traducción
> * [x]  En el archivo html "decide/dashboard/templates/dashboard/dashboard.html" faltan etiquetas de traducción

Los dos puntos referidos a record no son necesarios, puesto que son ficheros que se van a generar y sustituir cada vez que se descarguen las estadísticas. Veo oportuno no borrarlo para, si se requiere algún cambio en el mismo, ver de una manera más clara donde están localizados

JaimeBorregoConde committed:
fix: Fix reviewer found error
Added translation labels to html and one print in base/test has been eliminated

gonmarfer2 reviewed (approved):
Ya está todo bien
